### DISCUSSION TITLE
Add other languages translations support

### DISCUSSION THREAD

**Discussion by Elfocat** [RetroAchievements/RAWeb#1578] [Ideas]
Make possible to change the site language and be able to make the files in the _**lang**_ folder work natively

**Comment by wescopeland** [comment]
I would be interested in @luchaos' thoughts here. As it stands, even if we enable i18n there will be vast areas of the site that are untranslated with no support. There is also no current imperative to ensure new text/copy added to the site is i18n and l10n friendly.
